Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/TransmissionReverse.java

```diff
@@ -24,7 +24,7 @@ protected void initialize() {
 
     // Called repeatedly when this Command is scheduled to run
     protected void execute() {
-    	Robot.throwDaggersInBensEyes.reverse(null);
+//    	Robot.throwDaggersInBensEyes.reverse(null);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)